internal/resource-request-operator/resourceRequest_controller_methods.go

```diff
@@ -22,14 +23,20 @@ func requireTenantDeletion(resourceRequest *discoveryv1alpha1.ResourceRequest) b
 func (r *ResourceRequestReconciler) ensureTenant(ctx context.Context,
 	resourceRequest *discoveryv1alpha1.ResourceRequest) (requireUpdate bool, err error) {
 	remoteClusterID := resourceRequest.Spec.ClusterIdentity.ClusterID
-
+	annotationKey := fmt.Sprintf("%s-%s/%s", liqoconst.RemoteNamespaceAnnotationPrefix,
+		resourceRequest.Spec.ClusterIdentity.ClusterID, liqoconst.LiqoSuffix)
 	tenant := &capsulev1alpha1.Tenant{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: fmt.Sprintf("tenant-%v", remoteClusterID),
 		},
 	}
 	_, err = controllerutil.CreateOrUpdate(ctx, r.Client, tenant, func() error {
 		tenant.Spec = capsulev1alpha1.TenantSpec{
+			NamespacesMetadata: capsulev1alpha1.AdditionalMetadata{
+				AdditionalAnnotations: map[string]string{
+					annotationKey: liqoconst.RemoteNamespaceAnnotationValue,
+				},
+			},
```

Comment:
In my opinion, it is clearer to use a fixed key, like `liqo.io/remote-namespace` and set the clusterID as a value. What's your opinion @palexster ?

Reply:
What about remote-namespace/liqo.io = clusterID ?

Reply:
Usually, labels and annotation are in the form domain/name

Reply:
Totally agree